src/client.toit

```diff
@@ -115,25 +115,36 @@ class Client:
     certificate_ = certificate
 
   /**
-  Creates a new request for the given URI, of the form
-    "http://www.example.com:1080/path/to/file#fragment" using the given method.
+  Creates a new request for $path on the given server ($host, $port) using the given method.
 
   The $method is usually one of $GET, $POST, $PUT, $DELETE.
 
+  Instead of specifying $host and $path, you can specify a $uri, of the form
+    "http://www.example.com:1080/path/to/file#fragment".
+
   The returned $Request should be sent with $Request.send.
 
   The connection is automatically closed when the response's body ($Response.body) is
     completely read.
   */
-  new_request method/string --uri/string --headers/Headers=Headers -> Request:
-    parsed := ParsedUri_.parse uri
+  new_request method/string -> Request
```

Comment:
I would prefer splitting this into two functions.
One that takes a `--uri` and another that takes `--host`, `--port` and `--path`.